### DISCUSSION TITLE
Qt Creator blurry after installing Hyprland 0.34

### DISCUSSION THREAD

**Discussion by ATM-Jahid** [hyprwm/Hyprland#4351] [Q&A] [answered]
After updating to 0.34, everything in Qt Creator became blurry. Right after opening Qt Creator, everything is crispy. However, things become blurry a moment later.

I might have missed any related changes in Hyprland. It would be helpful to know how to solve this problem. Thanks.

**Comment by vaxerski** [comment]
is it running in xwl

**Reply by ATM-Jahid** [reply]
No, it is not.

**Reply by vaxerski** [reply, marked as answer]
try on git

**Reply by ATM-Jahid** [reply]
Tried on git. It works.
